container_api/src/lib.rs

```diff
@@ -44,7 +44,7 @@
 /// #[derive(StructOpt, Debug)]
 /// #[structopt(name = "hcc")]
 /// struct Opt {
-///     /// Output file
+///     /// /// Path to the toml configuration file for the container
```

Comment:
```suggestion
///     /// Path to the toml configuration file for the container
```

remove extra comment